### ISSUE TITLE
Unit tests should pass in all locales

### ISSUE THREAD

**Issue by veita** [drewnoakes/metadata-extractor#117]
This issue is related to https://github.com/drewnoakes/metadata-extractor/issues/6.  It occurs in tags/2.8.1, and master (HEAD).

```
git clone https://github.com/drewnoakes/metadata-extractor.git
cd metadata-extractor
mvn install
```

fails with

```
[INFO] BUILD FAILURE
...
Failed tests: testGimpGreyscaleWithManyChunks(com.drew.imaging.png.PngMetadataReaderTest):
  expected:<[Tue] Jan 01 04:08:30 GMT...> but was :<[Di] Jan 01 04:08:30 GMT...>
```

To fix this issue you may replace in Tests/com/drew/imaging/png/PngMetadataReaderTest.java, line 102

``` Java
 SimpleDateFormat formatter = new SimpleDateFormat("EE MMM DD HH:mm:ss z yyyy");
```

with

``` Java
 SimpleDateFormat formatter = new SimpleDateFormat("EE MMM DD HH:mm:ss z yyyy", Locale.US);
```

Or use the culture-neutral standard time format for string comparisons

``` Java
 formatter = new SimpleDateFormat("YYYY-MM-dd'T'HH:mm:ss'Z'");
 formatter.setTimeZone(TimeZone.getTimeZone("Etc/UTC"));
```

**Comment by LZaruba** [comment]
Hi, this issue is probably solved by solving #160 and #162.
I should have added the pull request to this one instead of creating a new bug, sorry.

LZ

**Comment by drewnoakes** [comment]
I believe this is now taken care of.
